---
name: Omkar Marne (he/him)
image: headshots/OmkarMarne.jpg
description: Ph.D. student and teaching assistant
role: phd
aliases:
  - Omkar Marne
  - O. Marne
  - Marne, O.
  - Marne, Omkar
links:
  linkedin: omkar-marne-919592122
  github: root2omkar
---

# Omkar Marne

## Who am I?

I am a Ph.D. student in UNC Charlotte's Department of Bioinformatics and Genomics. I have a Master's in information technology and Bachelor in computer science. My interest in bioinformatics comes from its pivotal role in developing vaccines against coronavirus disease 2019 (COVID-19).

The severity and worldwide spread of the COVID-19 pandemic were met with the scientific community's heroic efforts, including medicine, biology, public health, and computer science researchers. That effort led to the rapid development of several novel vaccines. Bioinformatics played a key role in producing the new COVID-19 vaccines in record time by helping to process large datasets from multiple disciplines. I am inspired by what bioinformaticists accomplished during the COVID-19 pandemic, and their efforts serve as a great source of inspiration and motivation.

I am interested in using my background in computer science to advance the field of bioinformatics and genomics, solving practical and urgent medical problems and improving healthcare.

## What am I working on?

Most phylogenetic analyses of SARS-CoV-2 fail to meet the standards of the field. For example, several phylogenies do not have an outgroup, making it impossible to determine the tree's root and the order of character transformations. As a second example, SARS-CoV-2 trees are often based on the spike (S) gene or whole genome alignments, which can obscure significant gene differences.

Our research aims to address these limitations by creating a phylogenetic tree of SARS-CoV-2 using outgroup sequences and individual gene alignments based on thoroughly annotated genomes. We will develop machine learning applications (decision trees) that classify SARS-CoV-2 lineages based on individual gene sequences. These applications will also be able to make predictions about evolutionary models unique to each gene or lineage.

Our analyses will be the backbone for future studies of SARS-CoV-2 mutation rates. By collaborating with other labs, we will be able to track the evolution of the virus and identify potential mutations that could affect its transmissibility or virulence.

